src/app/shop/[category]/[productId]/_components/TabContents/Review/ProductReviewList.tsx

```diff
@@ -0,0 +1,39 @@
+import ReviewItem from '@/components/ReviewItem/ReviewItem';
+import { NoReviewIcon } from '@/public/index';
+import type { ProductReviewType } from '@/types/ProductReviewTypes';
+import classNames from 'classnames/bind';
+import React from 'react';
+import styles from './ProductReviewList.module.scss';
+import ReviewImageList from './ReviewImageList';
+import ReviewPreview from './ReviewPreview';
+
+const cn = classNames.bind(styles);
+
+interface ProductReviewListProps {
+  data: ProductReviewType;
+}
+
+export default React.memo(function ProductReviewList({ data }: ProductReviewListProps) {
```

Comment:
```suggestion

export default memo(function ProductReviewList({ data }: ProductReviewListProps) {
```
import 로 사용가능한가요 혹시?